<?xml version="1.0"?>
<launch>
  <!-- 参数配置 -->
  <arg name="serial_port" default="/dev/ttyTHS0" />
  <arg name="baud_rate" default="460800" />
  <arg name="rate" default="1.0" />

  <!-- 启动状态信息发布器节点 -->
  <node pkg="limo_py_drive" type="limo_status_publisher.py" name="limo_status_publisher" output="screen">
    <param name="serial_port" value="$(arg serial_port)" />
    <param name="baud_rate" value="$(arg baud_rate)" />
    <param name="rate" value="$(arg rate)" />
  </node>
</launch>
